Packs/HostIo/Integrations/HostIo/hostio.py

```diff
@@ -0,0 +1,213 @@
+from typing import List, Any
+import urllib3
+from CommonServerPython import *
+from math import ceil
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+''' CONSTANTS '''
+
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+
+    def get_domain_data(self, domain: str) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'full/{domain}',
+            params={}
+        )
+
+    def get_search_data(self, field: str, value: str, limit: int, page: int) -> Dict[str, Any]:
+        return self._http_request(
+            method='GET',
+            url_suffix=f'domains/{field}/{value}',
+            params={'limit': limit,
+                    'page': page}
+        )
+
+    def test_module(self):
+        return self._http_request(
+            method='GET',
+            url_suffix='domains/ip/8.8.8.8',
+            params={}
+        )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module_command(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param client: the base client
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    try:
+        client.test_module()
+    except DemistoException as e:
+        if 'Forbidden' in str(e):
+            return 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
```

Comment:
raising the exception object again will reset the traceback to this location and mask the actual error location.
using an empty `raise` statement inside an `except` scope will raise the original exception and keep the traceback as expected.
```suggestion
            raise
```